systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PersistentVolumeClaimUtils.java

```diff
@@ -66,12 +67,24 @@ public static void waitUntilPVCAnnotationChange(String namespaceName, String clu
         LOGGER.info("PVC annotation has changed {}", newAnnotation.toString());
     }
 
-    public static void waitForPVCDeletion(String namespaceName, int volumesCount, JbodStorage jbodStorage, String clusterName) {
+    public static void waitForPVCDeletion(String namespaceName, String pvcName) {
+        TestUtils.waitFor("Wait for PVC deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(3).toMillis(), () -> {
+            if (kubeClient(namespaceName).getPVC(namespaceName, pvcName) == null) {
+                return true;
+            } else {
+                LOGGER.warn("PVC {} is not deleted yet! Triggering force delete by cmd client!", pvcName);
+                cmdKubeClient(namespaceName).deleteByName("pvc", pvcName);
+                return false;
```

Comment:
Do we need to trigger this operation multiple times? :) Are there some scenarios, where PVCs are stuck or why do need it? :)

Reply:
do we have this in "force" delete command `--force --grace-period=0 ` ? It might speed up the deletion a lot

Reply:
Anyway, it should be part of different PR, because it could influence much more parts than current changes